src/index.ts

```diff
@@ -23,6 +23,7 @@ export { default as issueVerifiableCredential } from "./issue/issue";
 export type { Iri, JsonLd, VerifiableCredential } from "./common/common";
 export {
   isVerifiableCredential,
+  isVerifiablePresentation,
```

Comment:
this function looks like it has an open TODO for adding some type checking around the VP proof. Do you think it would be reasonable to add that with these changes?

Reply:
I think this makes sense to be added to a different bit of work, where we actually prove the VP (and thus verify that the response is a proper signed VP)